pkg/util/net/ip/ip_utils.go

```diff
@@ -0,0 +1,67 @@
+package util
+
+import (
+	"bufio"
+	"net"
+	"os"
+)
+
+const DISABLE_IPV6_PATH = "/proc/sys/net/ipv6/conf/default/disable_ipv6"
+const IPV4_BIND_ADDRESS = "0.0.0.0"
+const IPV6_BIND_ADDRESS = "[::]"
+const IPV4_LOOPBACK_ADDRESS = "127.0.0.1"
+const IPV6_LOOPBACK_ADDRESS = "[::1]"
+
+// GetIPBindAddress returns IP bind address (either 0.0.0.0 or [::] according sysctl disable_ipv6)
+func GetIPBindAddress() string {
+	var disableIPv6 string
+	file, err := os.Open(DISABLE_IPV6_PATH)
```

Comment:
What is the reason you are not using exec.Command("sysctl", "net.ipv6.conf.default.disable_ipv6").CombinedOutput()?
(I remember we talked about it, so I guess there is a reason).

Reply:
Vladik suggested to use file open as other sysfs files are dealt with in the project, moreover it doesn't spawn a thread, travis is sensitive for threads in its goveralls,
and also its lighter.